samples-generator/lib/createGltf.js

```diff
@@ -1,18 +1,11 @@
 'use strict';
```

Comment:
The gltf/glb code path should be naming the batch id vertex attribute `_BATCHID_0`, not `_BATCHID`